crates/ruma-events/src/lib.rs

```diff
@@ -159,6 +159,8 @@ pub mod key;
 pub mod location;
 #[cfg(feature = "unstable-msc2867")]
 pub mod marked_unread;
+#[cfg(feature = "unstable-msc3401")]
+pub mod matrix_rtc;
```

Comment:
The events are organized according to the namespacing of their type, not in arbitrary modules. Please put everything back into the `call` module. If you want to split the event types definitions in several modules use submodules of the event's module.